### ISSUE TITLE
Callback is not invoked in MediaSession.onSessionTerminate

### ISSUE THREAD

**Issue by arudnev** [legastero/stanza.io#315]
I'm wondering if the intention was to call `super.onSessionTerminate` instead of `super.end` here:
 https://github.com/legastero/stanza.io/blob/0283e6e02790580ba3937069d533da61b1d282b9/src/jingle/MediaSession.js#L234

It seems that `super.onSessionTerminate` has correct general flow, including call to `cb`:

https://github.com/legastero/stanza.io/blob/0283e6e02790580ba3937069d533da61b1d282b9/src/jingle/ICESession.js#L153

Currently, `cb` is passed into `super.end` instead of `silent` parameter, so the callback is never invoked in processing of `session-terminate`.

**Comment by arudnev** [comment]
Yeah, it seems that with the following workaround the client now sends result in response to session-terminate, something that was missing before.

```javascript
MediaSession.prototype.onSessionTerminate = function(changes, cb) {
  for (const receiver of this.pc.getReceivers()) {
    this.onRemoveTrack(receiver.track);
  }
  ICESession.prototype.onSessionTerminate.call(this, changes, cb);
};
```

**Comment by legastero** [comment]
Fixed in 10.2.5.

I looked through and didn't find any other `super` calls that could cause further issues like this.
